### DISCUSSION TITLE
VPN and RDP

### DISCUSSION THREAD

**Discussion by emersonnpe** [Ylianst/MeshCentral#6090] [General]
We use Meshcentral in our organization, but we have a problem. When providing remote support to a user who is using a VPN (he is at home using a machine outside the domain and accessing his organization's machine via VPN and RDP) we have to perform tasks as an administrator, such as installing software. When trying to perform this action, the screen is blocked for the technician, but for the user, the box appears to enter the elevated user information (login and password). This box should appear for the technician, but the screen is blocked, and the user has to close the dialog box so that we can view his screen again.

Has anyone experienced this and knows how it can be resolved?

**Comment by si458** [comment]
the blocked screen thing only happens if you are using the `MeshCentralAssistant` as it runs as the logged in user rather then System
make sure you have installed the MeshAgent and its running as the System account on windows computers
